mesheryctl/pkg/utils/fixtures/platform/original/TestChangeEndpointConfig.yaml

```diff
@@ -1,29 +1,29 @@
 contexts:
   gke:
     components:
-      - meshery-istio
+    - meshery-istio
     channel: stable
     endpoint: http://localhost:9081
     platform: gke
     token: default
-    version: v0.4.2
+    version: latest
   local:
     endpoint: http://localhost:9081
     token: default
     platform: docker
     components:
-      - meshery-istio
+    - meshery-istio
     channel: stable
     version: v0.5.10
```

Comment:
perhaps, we're in need of identifying the last stable release's version number. In the case of meshery-istio, that currently would be v0.7.0.